style.css

```diff
@@ -23,7 +17,6 @@ body {
 
 /* Background Page */
 #section-1 {
```

Comment:
- [ ] There is an issue with your headline section background because it's showing the image in small size and it does not match the Figma design. I would suggest updating the image to make your portfolio look even better :raised_hands:

  - To download an image from Figma you have to select one piece of the background, e.g., one ellipse. And in the left side you will see the vector selected, if you go to the parent element you will see that it contains/select all the vectors in one piece/image, so you don't have to build the background one by one :wink:
  ![image](https://user-images.githubusercontent.com/81522435/144547584-c3c6b1fb-e536-4a24-b089-730b98540c08.png)

  - When I select the parent element/container (in this situation the "Header bg") you will see it will select all the small vectors :wink: and after this, you can select "export" on the right side of the page and download the image :smiley:
  ![image](https://user-images.githubusercontent.com/81522435/144547804-0175ef05-2e72-402d-956f-b41f1afdc5ef.png)

  - When replacing this background image instead of the vectors don't forget to use the property `background-size: cover` so your image takes all the section width :wink:

  I hope this helps :raised_hands:

Reply:
Wow, @alonsofl thank you so much, in the mobile version I built the bg one by one, it took me too much time. Amazing tip. I really appreciate it.👍